src/main/java/net/formio/validation/validators/AbstractValidator.java

```diff
@@ -48,6 +48,18 @@ public InterpolatedMessage warning(String elementName, String messageKey, Arg ..
 	public InterpolatedMessage info(String elementName, String messageKey, Arg ... args) {
 		return message(elementName, Severity.INFO, messageKey, null, args);
 	}
+
+	public InterpolatedMessage error(String elementName, String messageKey, String messageText, Arg ... args) {
```

Comment:
Přijde mi logičtější pořadí

`
(String elementName, String messageText, String messageKey, Arg ... args)
`

protože messageKey a args patří k sobě, ne?

Reply:
Ok upravím to

Reply:
upraveno

Reply:
Ok, v tomhle pořadí je to lepší.